packages/react-composites/tests/browser/call/hermetic/UnsupportedBrowserPage.test.ts

```diff
@@ -10,16 +11,22 @@ import { buildUrlWithMockAdapter, defaultMockCallAdapterState, test } from './fi
 /* @conditional-compile-remove(unsupported-browser) */
 test.describe('unsupportedBrowser page tests', async () => {
   test('unsupportedBrowser displays correctly without a help link', async ({ page, serverUrl }) => {
-    await page.goto(buildUrlWithMockAdapter(serverUrl, defaultMockUnsupportedBrowserPageState()));
+    const state = defaultMockUnsupportedBrowserPageState();
+    state.environmentInfo = setMockEnvironmentInfo(true, false, false);
```

Comment:
avoid many unnamed arguments particularly when they are booleans, reading this line what is `true, false, false`?
Instead use named params:
```
setMockEnvironmentInfo({ wellNamedArg1: true, wellNamedArg2: false, wellNamedArg3: false });
```